multisite-directory.php

```diff
@@ -49,7 +50,9 @@ public static function register () {
         add_action('delete_blog', array(__CLASS__, 'delete_blog'), 10, 2);
         add_action('network_admin_menu', array('WP_Multisite_Directory_Admin', 'network_admin_menu'));
         add_action('signup_blogform', array(__CLASS__, 'signup_blogform'));
-        add_action('add_signup_meta', array(__CLASS__, 'add_signup_meta'));
+        add_action('network_site_new_form', array(__CLASS__, 'signup_blogform'));
```

Comment:
About the bullets, I just realized that `wp_terms_checklist` doesn't include a `<ul>` in the code it returns, so we could just add a `<ul style="list-style:none;">` to the HTML  output.